packages/captp/test/engine-gc.js

```diff
@@ -0,0 +1,16 @@
+import v8 from 'v8';
```

Comment:
We do have an Ava clone for XS: https://github.com/Agoric/agoric-sdk/blob/3e4d66428e2b4cad3ab65045b805e20ff7cc746c/packages/xsnap/src/avaXS.js

Not saying this PR needs to make an XS test too, but it would be helpful for this module to document why this isn't done and perhaps what it would take to pull it off.

Reply:
Done and pushed.